x-pack/solutions/search/plugins/search_indices/public/components/index_mangement/app.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ChromeBreadcrumb, ScopedHistory } from '@kbn/core/public';
+import { IndexManagementPluginSetup } from '@kbn/index-management-shared-types';
+import { reactRouterNavigate } from '@kbn/kibana-react-plugin/public';
+import { KibanaPageTemplate } from '@kbn/shared-ux-page-kibana-template';
+import React, { useRef, useEffect, useCallback } from 'react';
+import { i18n } from '@kbn/i18n';
+import { useKibana } from '../../hooks/use_kibana';
+
+const breadcrumbText = i18n.translate('xpack.searchIndices.breadcrumb', {
+  defaultMessage: 'Content',
+});
+
+const breadcrumb = {
+  text: breadcrumbText,
+};
+
+export const IndexManagementApp: React.FC<{ indexManagement: IndexManagementPluginSetup }> = ({
+  indexManagement,
+}) => {
+  const { history, searchNavigation, cloud } = useKibana().services;
+  const indexManagementRef = useRef(null);
+  const setBreadcrumbs = useCallback(
+    (crumbs: ChromeBreadcrumb[] = [], appHistory?: ScopedHistory) => {
+      const wrapBreadcrumb = (item: ChromeBreadcrumb, scopedHistory: ScopedHistory) => ({
+        ...item,
+        ...(item.href ? reactRouterNavigate(scopedHistory, item.href) : {}),
+      });
+
+      const wrapBreadcrumbValue = !cloud?.isServerlessEnabled
+        ? [wrapBreadcrumb(breadcrumb, history)]
+        : [];
+
+      const breadcrumbValue = [
+        ...wrapBreadcrumbValue,
+        ...crumbs.map((item) => wrapBreadcrumb(item, appHistory || history)),
+      ];
+
+      searchNavigation?.breadcrumbs.setSearchBreadCrumbs(breadcrumbValue);
+    },
+    [searchNavigation, history, cloud]
+  );
+
+  useEffect(() => {
+    // mount
+    let indexManagementUnmount: () => void | undefined;
+    indexManagement
+      .indexManagementApp({
+        element: indexManagementRef.current!,
+        setBreadcrumbs,
+        history,
+      })
```

Comment:
We not use `!`s 

```suggestion
    if (indexManagementRef.current === null) return;
    // mount
    let indexManagementUnmount: () => void | undefined;
    indexManagement
      .indexManagementApp({
        element: indexManagementRef.current,
        setBreadcrumbs,
        history,
      })
```

Reply:
update https://github.com/elastic/kibana/pull/209599/commits/32539a5559d12cc5e3db4f12829288eae457a312